src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,72 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+/* eslint-disable import/no-extraneous-dependencies */
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
+
+export const fetchRockets = createAsyncThunk('books/fetchBooks', async () => {
```

Comment:
- [ Optional Change ]
- [ ] As much as we're free to name our files, it's a good practice to name our files, variable names, functions, and actions as meaningful as what they actually do. Here, you're fetching rockets but you name your thunk action as books/fetchBooks. Kindly modify it 👍🏽